docker/hooks/post_build

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# This hook is called with the current directory set to the same as the Dockerfile, so we go back
+# to top level.
+cd ..
+
+# We always build an image containing nginx and zipkin-lens in addition to the default image of zipkin-server
+echo Building zipkin-ui
+docker build -f $DOCKERFILE_PATH -t openzipkin/zipkin-ui:$DOCKER_TAG --target zipkin-ui .
+
+if [[ "$DOCKER_TAG" == "master" ]]; then
```

Comment:
Should we move this to the build hook before the image build? That way the build always starts with an up to date image

Reply:
The first build command goes through all steps in order, so it will have built this and the lens image as intermediate steps. We have to run build again for them so they get tagged, but as the build has happened already these docker build commands in the `post_build` are all served from cache and don't take any time.

So the order of building these doesn't matter so much, but I think it's a bit clearer that both of these are secondary artifacts by having them in `post_build`